### PR TITLE
In 388 add endpoint to add person using legacy

### PR DESCRIPTION
**Pull request by saraOSI**
### Jira Story

- [Add API endpoint to create person on a screening using only legacy id INT-388](https://osi-cwds.atlassian.net/browse/INT-388)

### Purpose
We need to separate creating a person from searching for a person. Toward that end, the front end will search for people directly against Dora using a lighter elastic search index. Then, when a user adds a person to a screening, the only element in the post request will be the legacy descriptor. The API will then search agains the full elastic search index to get all of the data needed to add a legacy person to a screening.

### Background
Eventually, this functionality should be replaced with a person service endpoint.

### CODE REVIEW

File: spec/requests/api/v1/screenings/screening_people_spec.rb [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+        end
+      end
+
+      context 'without permission to add sensitive' do
+        let(:jwt_token) { { staffId: '123', privileges: [] } }
+
+        context 'with sensitive participant' do
+          let(:dora_search_result) { { sensitivity_indicator: 'S' } }
+
+          it 'creates a participant' do
```
Comment (ThomasOSI) [inline spec/requests/api/v1/screenings/screening_people_spec.rb]: Should be "does not create a participant"?

File: app/controllers/api/v1/screening_people_controller.rb [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+          legacy_descriptor: legacy_descriptor_params[:legacy_descriptor],
+          screening_id: params[:screening_id],
+          can_create_sensitive: current_user_has_permission(:create, :add_sensitive_people),
+          security_token: security_token
+        )
+        render json: participant, status: :created
+      end
+
+      def legacy_descriptor_params
+        params.permit(legacy_descriptor: %i[legacy_id legacy_table_name])
```
Comment (jgedeon-osi) [inline app/controllers/api/v1/screening_people_controller.rb]: These are the only two legacy_descriptor values we care about?

File: app/services/create_screening_person.rb [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+
+  private
+
+  def new_person_params(screening_id)
+    { screening_id: screening_id }
+  end
+
+  def legacy_person_params(security_token, legacy_id, screening_id)
+    find_person(security_token, legacy_id)
+    person_search_result.merge(
```
Comment (ThomasOSI) [inline app/services/create_screening_person.rb]: From a unit test perspective, do we care about all the fields?

File: spec/requests/api/v1/screenings/screening_people_spec.rb (hunk 1 above)
Comment (saraOSI) [reply spec/requests/api/v1/screenings/screening_people_spec.rb]: Yep! Thanks for catching that.

**Pull request merged**